tests/test_plugin.py

```diff
@@ -2981,6 +2982,28 @@ def test_commando_listrunes(node_factory):
     assert not_our_rune['stored'] is False
     assert not_our_rune['our_rune'] is False
 
+def test_commando_blacklist(node_factory):
+    l1 = node_factory.get_node()
+    l1.rpc.commando_rune()
+    l1.rpc.commando_rune()
```

Comment:
Since we assume this below, perhaps change these to `assert l1.rpc.commando_rune()['unique_id'] == 1` etc?